modules/install/pages/install-environments.adoc

```diff
@@ -6,11 +6,11 @@
 
 == Java Runtime Environment (JRE) — Analytics Service Only
 
-The Analytics Service requires a Java Runtime Environment to be installed. Only https://openjdk.java.net/groups/hotspot/[HotSpot]-based JVMs, which includes the ones provided by OpenJDK and Oracle's JDK, are supported.
+The Analytics Service requires a Java Runtime Environment to be installed. Only https://openjdk.java.net/groups/hotspot/[HotSpot-based JVMs], which includes the ones provided by OpenJDK and Oracle's JDK, are supported.
```

Comment:
Seems like a good candidate for an external link icon.

```suggestion
https://openjdk.java.net/groups/hotspot/[HotSpot-based JVMs^]
```

Reply:
Yes, you're right. Good spot.